examples/mobile/HomeApp/js/app.js

```diff
@@ -9,7 +9,19 @@
 		},
 		// TODO: remove below directive after webClipList use
 		// eslint-disable-next-line no-unused-vars
-		webClipList = [];
+		webClipList = [
```

Comment:
I have a question.
It looks you pre-defined the webClipList.
If I remove webclip (example "rv-remote-control"), is it also update??

Reply:
webClipList will be updated on application start -
loadWebClipList() updates it basing on the api/webcliplist